data/README.md

```diff
@@ -1,132 +1,199 @@
+		startSale() {
+			return {
+				type: 'START_SALE',
+			};
+		},
+	},
 
-If your module or plugin needs to store and manipulate client-side data, you'll have to register a "reducer" to do so. A reducer is a function taking the previous `state` and `action` and returns an update `state`. You can learn more about reducers on the [Redux Documentation](https://redux.js.org/docs/basics/Reducers.html)
+	selectors: {
+		getPrice( item ) {
+			const price = state.prices[ item ];
 
-This function takes two arguments: a `key` to identify the module (example: `myAwesomePlugin`) and the reducer function. It returns a [Redux-like store object](https://redux.js.org/docs/basics/Store.html) with the following methods:
+			if ( state.sale ) {
+				return price * 0.8;
+			}
+
+			return price;
+		},
+	},
+} );
+```
 
-#### `store.getState()`
+A [**reducer**](https://redux.js.org/docs/basics/Reducers.html) is a function accepting the previous `state` and `action` as arguments and returns an updated `state` value.
 
-Returns the [state object](https://redux.js.org/docs/Glossary.html#state) of the registered reducer. See: https://redux.js.org/docs/api/Store.html#getState
+The **`actions`** object should describe all [action creators](https://redux.js.org/glossary#action-creator) available for your store. An action creator is a function that optionally accepts arguments and returns an action object to dispatch to the registered reducer. _Dispatching actions is the primary mechanism for making changes to your state._
 
-#### `store.subscribe( listener: function )`
+The **`selectors`** object includes a set of functions for accessing and deriving state values. A selector is a function which accepts state and optional arguments and returns some value from state. _Calling selectors is the primary mechanism for retrieving data from your state_, and serve as a useful abstraction over the raw data which is typically more susceptible to change and less readily usable as a [normalized object](https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape#designing-a-normalized-state).
 
-Registers a [`listener`](https://redux.js.org/docs/api/Store.html#subscribe) function called everytime the state is updated.
+The return value of `registerStore` is a [Redux-like store object](https://redux.js.org/docs/basics/Store.html) with the following methods:
 
-#### `store.dispatch( action: object )`
+- `store.getState()`: Returns the state value of the registered reducer
+   - _Redux parallel:_ [`getState`](https://redux.js.org/api-reference/store#getState)
+- `store.subscribe( listener: Function )`: Registers a function called any time the value of state changes.
+   - _Redux parallel:_ [`subscribe`](https://redux.js.org/api-reference/store#subscribe(listener))
+   - Differences from Redux:
+      - In Redux, a subscriber is called on every dispatch, regardless of whether a change has occurred. In `@wordpress/data`, a subscriber is only called when state has changed.
```

Comment:
Not certain we have to mention this as it's a very small difference.

Reply:
>Not certain we have to mention this as it's a very small difference.

Right. One of my initial goals in writing this was to highlight any and all differences to Redux, but I agree for most people this would have no impact. Do you think we should mention this anywhere? The "Comparison with Redux" section perhaps?

Reply:
Moving to "comparison with redux" sounds good to me